pandas/util/testing.py

```diff
@@ -2826,3 +2826,28 @@ def skipna_wrapper(x):
             return alternative(nona)
 
     return skipna_wrapper
+
+
+def _get_cython_table_params(ndframe, func_names_and_expected):
```

Comment:
hmm didn't realize you are actually importing from conftest, so on 2nd though, move this to ``pandas.conftest`` and import from there (you have to explicity import non-fixtures FYI)